README.md

```diff
@@ -1,2 +1,45 @@
 # wifi-sensor
+
 A sensor that detects wifi strength
+
+## Usage
+
+### 1. Build binary
+
+If you clone this repository to the target environment where you run your Viam robot, then you can build a binary named `wifi` with:
+
+```
+go build -o wifi
+```
+
+Alternatively, if you want to build a binary for a different target environment, please use the [viam canon tool](https://github.com/viamrobotics/canon).
+
+### 2. Add to robot configuration
+
+Copy the binary to the robot (system where viam-server is running) and add the following to your configuration:
+
+```
+  ...
+  "modules": [
+    ...,
+    {
+      "executable_path": "<path_to_binary>",
+      "name": "wifisensor"
+    },
+    ...,
+  ],
+  "components": [
+    ...,
+    {
+      "name": "wifi",
+      "type": "sensor",
+      "model": "viam-labs:sensor:linux-wifi",
+      "depends_on": [],
+      "attributes": {}
```

Comment:
[nit] I think these could just be left out entirely, but no harm with them here either.